docs/contributing.rst

```diff
@@ -102,6 +102,11 @@ You can watch the source directory and run JupyterLab at the same time in differ
 
 With the watch command running, every saved change will immediately be built locally and available in your running JupyterLab. Refresh JupyterLab to load the change in your browser (you may need to wait several seconds for the extension to be rebuilt).
 
+.. note::
+
+   ``jlpm run watch`` will sit and wait for a change once started. Edit a file
+   to trigger a build.
+
```

Comment:
This was true before this PR, but noticed it again during testing and figured it deserved a callout in the docs since it's a bit different behavior from most other "watch" tools I've used.